internal/services/oracle/oracle_exadata_infrastructure_resource_test.go

```diff
@@ -53,6 +53,9 @@ func TestExaInfra_complete(t *testing.T) {
 			Config: r.complete(data),
 			Check: acceptance.ComposeTestCheckFunc(
 				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("shape").HasValue("Exadata.X11M"),
+				check.That(data.ResourceName).Key("database_server_type").HasValue("X11M"),
+				check.That(data.ResourceName).Key("storage_server_type").HasValue("X11M-HC"),
```

Comment:
These should be unnecessary as the import check will validate that all values match those in the config on read.
```suggestion
```